src/components/DataGrid.jsx

```diff
@@ -404,12 +404,26 @@ class DataGrid1 extends Component {
                   <img src="../../react-logo-1.png" alt="logo" className="yjsg-logo-img"/>
                 </div>
                 <h2 className="student-info-heading">{yjsgHeader}</h2>
-                <div className={'logoutButtonContainer'}>
+                <div className={'logoutButtonContainer display-mobile-none'}>
```

Comment:
```suggestion
                <div className="logout-button-container display-mobile-none">
```

Follow the CSS class names convention.

Reply:
done